### ISSUE TITLE
[Bug] Support for mocha v9

### ISSUE THREAD

**Issue by vinayakkulkarni** [sysgears/mochapack#106]
**Describe the bug**

When I try to install `mochapack`, npm throws error

**Screenshots**

```sh
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: tcl-lineofsight-frontend@3.2.1
npm ERR! Found: mocha@9.0.0
npm ERR! node_modules/mocha
npm ERR!   dev mocha@"^9.0.0" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer mocha@">=6 <9" from mochapack@2.1.2
npm ERR! node_modules/mochapack
npm ERR!   dev mochapack@"^2.1.2" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR!
npm ERR! See /Users/vinayak/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/vinayak/.npm/_logs/2021-06-08T05_19_22_481Z-debug.log
```

**Environment if relevant (please complete the following information):**

 - OS: macOS Big Sur
 - Node version 16.3.0
 - Mochapack version 2.1.2
 - Webpack version 4

**Additional context**

Mocha released [v9](https://github.com/mochajs/mocha/releases/tag/v9.0.0) few hours ago.

**Comment by GiantVlad** [comment]
@vinayakkulkarni  try to change "mocha": "^8.4.0" in your package.json

**Comment by larixer** [comment]
Fixed via: #124 
Published in `mochapack@2.1.4`
CC: @BeeMargarida @GiantVlad
